src/storage/double_write_buffer.c

```diff
@@ -717,81 +719,86 @@ dwb_destroy_wait_queue (DWB_WAIT_QUEUE * wait_queue, pthread_mutex_t * mutex)
 }
 
 /*
- * dwb_adjust_write_buffer_values () - Adjust double write buffer values.
+ * dwb_adjust_value () - Adjust value to power of 2.
```

Comment:
I want it to be more meaningful so that it says the purpose or role of this function. For example, It can be `dwb_adjust_[something]_value()`, `dwb_power2_ceil()` or something. I suppose you can make a better choice than me.